pkg/packages.go

```diff
@@ -76,7 +76,7 @@ func Ensure(direct spec.JsonnetFile, vendorDir string, oldLocks map[string]deps.
 
 	// remove them
 	for _, dir := range names {
-		name := strings.TrimPrefix(dir, "vendor/")
+		name := strings.TrimPrefix(dir, vendorDir+"/")
```

Comment:
I *think* `filepath.Rel` works better here:

```suggestion
		name := filepath.Rel(vendorDir, dir)
```